.prettierrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
do we need to create our own base config for these things 🙈 we're setting up prettier + eslint + typescript quite often.

Reply:
yup. we should. didn't know that you left this comment and I told you the same thing in the evening. Let's find a easy/clean way to share these configs